dmenufm-action

```diff
@@ -564,7 +564,7 @@ FM_EYE () {
 	eyeid="$(xprop -root _NET_ACTIVE_WINDOW | cut -d ' ' -f 5)"
 	[ -n "$HERE" ] && FileOpen "$HERE" &
     done
-    wmctrl -c :ACTIVE:
+    wmctrl -ic "$eyeid"
```

Comment:
Personally, I think I would still rather just use normal selection to open the file then close it. To be fair, this is mostly because I feel like I have a good idea of what I am looking at/editing before I look, so previewing it is not a priority for me. On the occasion I do select the wrong file, quitting the program normally is not too much of a hassle. I could see the use case for someone just browsing to see what everything is before choosing to actually open something.